powerhome/playbook
# [PLAY-24]  Improve max width global props docs/visual-guidelines

#### Screens

<img width="618" alt="Screen Shot 2021-11-17 at 10 58 22 AM" src="https://user-images.githubusercontent.com/84339183/142235312-da7f36d1-2280-49b1-a624-9b1045fd0f0b.png">

#### Breaking Changes

No breaking changes

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-24

#### How to test this
Go to: https://playbook.powerapp.cloud/visual_guidelines and make sure the max width example is there.

#### Checklist:
